faucet/src/main.rs

```diff
@@ -112,22 +113,28 @@ async fn request_gas(
     Extension(state): Extension<Arc<AppState>>,
 ) -> impl IntoResponse {
     let result = match payload {
-        FaucetRequest::FixedAmountRequest(requests) => {
+        FaucetRequest::FixedAmountRequest(request) => {
+            info!("Got a request: {:?}", request);
```

Comment:
Instead of interpolating the request into the log string, we should split it out as a separate parameter.  For example,

    info!(request =? request, "Got a request")

If the request has separate fields, it is better to split it out even more.  This makes filtering much easier.  For example, you could have a field for recipient.